packages/table/src/Cell/HeaderCell/HeaderCell.spec.tsx

```diff
@@ -6,7 +6,7 @@ import { axe } from 'jest-axe';
 import {
   getTestColumnsProps,
   useTestHookCall,
-} from '../../utils/testHookCalls';
+} from '../../utils/testHookCalls.testutils';
```

Comment:
intentional?

Reply:
Yeah, that's the only reliable way to tell depcheck to ignore certain files (without having a giant list of specific files)

Reply:
ah got it